docker/README.md

```diff
@@ -118,7 +118,7 @@ docker run -it --rm \
     --network "bk_network" \
     --env BK_zkServers=test_zookeeper:2181 \
     apache/bookkeeper \
-    bookkeeper shell metaformat
+    /opt/bookkeeper/bin/bookkeeper shell metaformat
```

Comment:
Do we need the `/opt/bookkeeper` prefix? @zymap Please help take a look, thanks.

Reply:
https://github.com/apache/bookkeeper/blob/1e3efa112b66aa597def7cc247a1ffa9e7732301/docker/Dockerfile#L59
`mv bookkeeper-server-${BK_VERSION}/ /opt/bookkeeper/`

https://github.com/apache/bookkeeper/blob/1e3efa112b66aa597def7cc247a1ffa9e7732301/docker/Dockerfile#L68

https://github.com/apache/bookkeeper/blob/1e3efa112b66aa597def7cc247a1ffa9e7732301/docker/scripts/entrypoint.sh#L58